core/server/data/fixtures/index.js

```diff
@@ -156,13 +157,81 @@ to003 = function () {
     });
 };
 
+/**
+ * Update ghost_foot to include a CDN of jquery if the DB is migrating from
+ * @return {Promise}
+ */
+to004 = function () {
+    var value,
+        ops = [],
+        upgradeOp,
+        jquery = [
+            '<!-- only delete this line if you are ABSOLUTELY SURE your theme doesn\'t require jQuery -->\n',
+            '<script type="text/javascript" src="http://code.jquery.com/jquery-1.11.3.min.js"></script>\n\n'
+        ],
+        privacyMessage = [
+            'Because jQuery is no longer included for Ghost themes by default,',
+            'a link to the jQuery library via a CDN has been added to your code-injection setting,',
+            'If you wish to remove it for privacy reasons, please ensure it is not required by your theme.'
+        ];
+
+    // add jquery setting and privacy info
+    upgradeOp = models.Settings.findOne('ghost_foot').then(function (setting) {
+        if (setting) {
+            value = setting.attributes.value;
```

Comment:
Can we have a logInfo message added here so that there's output on the cli for 1) moving jQuery to ghost_foot and 2) if the notification is added?